client/favorites.jsx

```diff
@@ -1,23 +1,60 @@
 import React from 'react';
 import NavBar from './navBar';
 import Loader from './loader';
+import fetch from 'node-fetch';
 
-export default class SearchResults extends React.Component {
+
+export default class Favorites extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      isLoading: false
+      favoritesData: '',
+      isLoading: true,
+      removalId: []
+
     };
+    this.handleRemove = this.handleRemove.bind(this)
   }
 
+  handleRemove(e) {
+    e.preventDefault()
+    let removalId = ''
+    let copy = ''
+    const data = this.state.favoritesData
+    for (let i = 0; i < data.length; i ++) {
+      if (data[i].hotelId === Number(e.target.dataset.hotelId)) {
+        removalId = data[i].hotelId
+        copy = [...this.state.removalId]
+        copy.push(removalId)
+        break;
+      }
+    }
+
+    fetch(`/api/1/${removalId}`, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    })
+      .then(response => response.json())
```

Comment:
If you don't really care about the response body, you can skip this `response.json()` part.